windows/ReactTestApp/MainPage.xaml

```diff
@@ -8,21 +8,47 @@
     mc:Ignorable="d"
     xmlns:react="using:Microsoft.ReactNative">
 
+    <Page.Resources>
+        <Style TargetType="Button" x:Key="ReactMenuButton" BasedOn="{StaticResource TextBlockButtonStyle}">
+            <Setter Property="Foreground" Value="Black"/>
+            <Setter Property="FontSize" Value="12pt"/>
+        </Style>
+    </Page.Resources>
+
     <Grid>
         <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
             <RowDefinition Height="Auto"/>
             <RowDefinition Height="*"/>
         </Grid.RowDefinitions>
 
-        <MenuBar VerticalAlignment="Top" Grid.Row="0">
-            <MenuBarItem Title="React" x:Name="MenuFlyout">
-                <MenuFlyoutItem Text="Load from JS bundle" Click="LoadFromJSBundle"/>
-                <MenuFlyoutItem Text="Load from dev server" Click="LoadFromDevServer"/>
-                <MenuFlyoutSeparator/>
-            </MenuBarItem>
-        </MenuBar>
+        <Grid x:Name="TitleBar">
+            <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition x:Name="LeftPaddingColumn" Width="0"/>
+                    <ColumnDefinition Width="Auto"/>
+                    <ColumnDefinition/>
+                    <ColumnDefinition x:Name="RightPaddingColumn" Width="0"/>
+                </Grid.ColumnDefinitions>
+                <Button Grid.Column="1" Content="React" x:Name="ReactButton" Click="OpenReactMenu" Style="{StaticResource ReactMenuButton}">
+                    <Button.Flyout>
+                        <Flyout Placement="BottomEdgeAlignedLeft" x:Name="MenuFlyout">
+                            <StackPanel x:Name="ReactMenu">
+                                <Button Click="LoadFromJSBundle" Style="{StaticResource ReactMenuButton}">Load from JS bundle</Button>
+                                <Button Click="LoadFromDevServer" Style="{StaticResource ReactMenuButton}">Load from dev server</Button>
+                                <MenuFlyoutSeparator/>
+                            </StackPanel>
+                        </Flyout>
+                    </Button.Flyout>
+                </Button>
```

Comment:
Why can't we use `MenuBar` here? It seems that by mimicking it, we will always chase styling changes. Right now, it looks like the buttons have too much padding for instance.